### ISSUE TITLE
Permission denied: '/tmp/combo_locks/b3Zvc19jb25maWc=.lock'

### ISSUE THREAD

**Issue by goldyfruit** [OpenVoiceOS/ovos-installer#171]
OVOS doesn't start after reboot because of `Permission denied: '/tmp/combo_locks/b3Zvc19jb25maWc=.lock'`. This might be due to `ovos-phal-admin` service starting before the rest.

```
Oct 10 17:15:14 x270.home.lan ovos-core[5528]: Traceback (most recent call last):
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/bin/ovos-core", line 5, in <module>
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     from ovos_core.__main__ import main
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_core/__init__.py", line 15, in <module>
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     from ovos_config.config import Configuration
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/__init__.py", line 1, in <module>
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     from ovos_config.config import Configuration, LocalConf, RemoteConf
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/config.py", line 40, in <module>
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     class Configuration(dict):
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/config.py", line 44, in Configuration
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     default = MycroftDefaultConfig()
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:               ^^^^^^^^^^^^^^^^^^^^^^
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/models.py", line 180, in __init__
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     super().__init__(DEFAULT_CONFIG)
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/models.py", line 153, in __init__
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     super().__init__(path)
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/models.py", line 68, in __init__
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     self.load_local(path)
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/ovos_config/models.py", line 102, in load_local
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     with self.__lock:
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/combo_lock/combo_lock.py", line 92, in __enter__
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     self.acquire()
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/combo_lock/combo_lock.py", line 58, in acquire
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     result = self._acquire(blocking)
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:              ^^^^^^^^^^^^^^^^^^^^^^^
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/combo_lock/combo_lock.py", line 82, in _acquire
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     self.plock.acquire()
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/filelock/_api.py", line 332, in acquire
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     self._acquire()
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:   File "/home/goldyfruit/.venvs/ovos/lib64/python3.11/site-packages/filelock/_unix.py", line 42, in _acquire
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:     fd = os.open(self.lock_file, open_flags, self._context.mode)
Oct 10 17:15:14 x270.home.lan ovos-core[5528]:          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 10 17:15:14 x270.home.lan ovos-core[5528]: PermissionError: [Errno 13] Permission denied: '/tmp/combo_locks/b3Zvc19jb25maWc=.lock'
Oct 10 17:15:14 x270.home.lan systemd[4465]: ovos-core.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by JarbasAl** [comment]
wasn't that fixed here? https://github.com/forslund/combo-lock/issues/21 :thinking:

**Comment by goldyfruit** [comment]
https://github.com/OpenVoiceOS/ovos-installer/commit/750f2570105f32700421ab3250214817f52552f4

**Comment by goldyfruit** [comment]
> wasn't that fixed here? [forslund/combo-lock#21](https://github.com/forslund/combo-lock/issues/21) 🤔

`ovos-phal-admin` starts before the restart of the other services as this one is running as `root`. This is not a `combo_locks` issue.
